### PR TITLE
Channel change docs

### PR DESCRIPTION
**Pull request by Koenkk**
CC: @Nerivec

### CODE REVIEW

File: docs/advanced/zigbee/02_improve_network_range_and_stability.md [hunk 1]
```diff
@@ -28,7 +28,8 @@ RF connection between the adapter and other devices also depends on the way it i
 ![rotating USB connector](https://i.imgur.com/AI41Oxz.png)
 
 ## Reduce Wi-Fi interference by changing the Zigbee channel
-**Changing the Zigbee channel requires repairing of all your devices!**
+
+**Changing the Zigbee channel might require repairing some of your devices, read the [documentation for more info](../../guide/configuration/zigbee-network.md#changing-the-zigbee-channel)**
```
Comment (Nerivec) [inline docs/advanced/zigbee/02_improve_network_range_and_stability.md:32]: That `re-pairing` again 😁

File: docs/guide/configuration/zigbee-network.md [hunk 2]
```diff
@@ -25,7 +26,8 @@ advanced:
   pan_id: 0x1a62
   # Optional: Zigbee extended pan ID, GENERATE will make Zigbee2MQTT generate a new extended panID on next startup (default: shown below)
   ext_pan_id: [0xDD, 0xDD, 0xDD, 0xDD, 0xDD, 0xDD, 0xDD, 0xDD]
-  # Optional: ZigBee channel, changing requires re-pairing of all devices. (Note: use a ZLL channel: 11, 15, 20, or 25 to avoid Problems)
+  TODO
```
Comment (Nerivec) [inline docs/guide/configuration/zigbee-network.md:29]: TODO?

File: docs/guide/configuration/zigbee-network.md [hunk 3]
```diff
@@ -43,6 +45,26 @@ Set `network_key: GENERATE` to let Zigbee2MQTT generate a new random key on the
 channel](../../advanced/zigbee/02_improve_network_range_and_stability.html#reduce-wi-fi-interference-by-changing-the-zigbee-channel.md)
 :::
 
+### Changing the Zigbee channel
+
+Changing the channel of an existing Zigbee network is supported. In Zigbee, this is done by broadcasting a network update indicating the channel change. However, there is a caveat: devices that are asleep during the broadcast will not switch to the new channel and will thus become unreachable. These are typically battery-powered end devices, and the only way to move them to the new channel is by re-pairing them.
```
Comment (Nerivec) [inline docs/guide/configuration/zigbee-network.md:50]: That's not quite what I've observed (from my own tests and from users feedback). Sleepy switch just fine in general (after they wake up, they notice they are no longer connected and trigger a scan to rejoin).
The issue with devices not following seem to be more about not following spec than their power type.

The caveat to be noted would be that sleepy may only rejoin the next time they wake up, which, for devices like basic motion sensors, or door sensors, may not be until they are triggered again.
_Advice would be to go around the house a few minutes after the channel change is effective, to wake everything up._ 😁

File: docs/guide/configuration/zigbee-network.md (hunk 2 above)
Comment (Koenkk) [reply docs/guide/configuration/zigbee-network.md:29]: Whoops, forgot to remove it.

File: docs/guide/configuration/zigbee-network.md (hunk 3 above)
Comment (Koenkk) [reply docs/guide/configuration/zigbee-network.md]: Good to know, updated the docs.

**Pull request merged**